src/nools/task-emitter.js

```diff
@@ -163,4 +170,22 @@ function emitTasks(taskDefinition, Utils, Task, emit, c, r) {
   }
 }
 
+function defaultResolvedIf (c, r, event, dueDate, resolvingForm, Utils) {
+  var start = 0;
```

Comment:
Please use `cons` and `let` instead of `var`

Also here you can extract the resolvingForm and add validations. Remember to exit early to keep code flat and avoid unnecessary processing :)